python/llm/src/ipex_llm/transformers/convert.py

```diff
@@ -734,7 +734,14 @@ def _optimize_pre(model):
     # for qwen2
     if model.config.model_type == "qwen2":
         from ipex_llm.transformers.models.qwen2 import merge_qkv
-        model.apply(merge_qkv)
+        # Skip merge_qkv if quant_method is 'gptq'
+        should_apply_merge_qkv = (
+            not hasattr(model.config, "quantization_config") or
+            not hasattr(model.config.quantization_config, "quant_method") or
+            model.config.quantization_config.quant_method != "gptq"
+        )
+        if should_apply_merge_qkv:
+            model.apply(merge_qkv)
```

Comment:
Is gptq loaded as q4_1? If so, why can't we use `merge_qkv`? @qiuxin2012

Reply:
We need a new merge_qkv for gptq here. Just convert q k v to one LowBitLinear.

Reply:
> We need a new merge_qkv for gptq here. Just convert q k v to one LowBitLinear.
It's different from q4_1?

Reply:
gptq is loaded as q4_1. However, gptq's weights have already been quantized. If we want to use merge_qkv, we need to dequantize from the gptq format to the normal format first, perform merge_qkv, and then quantize back into the LowBitLinear. 

I will try this method.

Reply:
> gptq is loaded as q4_1. However, gptq's weights have already been quantized. If we want to use merge_qkv, we need to dequantize from the gptq format to the normal format first, perform merge_qkv, and then quantize back into the LowBitLinear.
> 
> I will try this method.

Instead of dequantization, I think we can just rearrange the quantized qkv tensors into a combined one? Anyway I think we may fix it later in a separate PR if needed.

Reply:
I'll merge this PR first so users can start using it, and then I'll submit another PR to address further fixes.

Reply:
I guess we probably cannot merge qkv if `quantization_config.desc_act==True`.